source/docs/articles/local/drush-versions.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Managing Drush Versions on Pantheon
+description: Learn about Pantheon's default Drush version and how to implement site-local usage.
+keywords: Drupal drush, command line, drupal, terminus drush, cli, site-local drush
+---
+By default, Pantheon runs Drush 8.x on newly created Drupal sites and 5.x on sites created prior to [date]. We highly recommend running Drush 8.x locally when interacting with Drupal sites on Pantheon, but 7.x is also supported. Alias files generated by Pantheon are not compatible with 6.x, however you can execute commands locally with this version by including the `--strict=0` option.
```

Comment:
Yes, it looks good. It might be clearer to re-order the middle sentence:

"When running Drush locally, we highly recommend running Drush version 8.x, although 7.x is also supported."

This makes it a little clearer that the "6.x" in the third sentence refers to running Drush 6.x on the local machine.  You can ask for Drush 6.x on the remote end with a policy file, but what you get is a modified-for-Pantheon-version of Drush 6.2.  Drush 6.6 is the last release on the 6.x line.  I've never touched this, because Drush 6 was never supported on the platform. Probably not worth documenting that; people who use Drush 6 on the Pantheon platform do so at their own risk. :)